components/TextArea/TextArea.tsx

```diff
@@ -65,15 +73,24 @@ const Root = styled.textarea<{
   resize: ${({ resizable }) => (resizable ? "vertical" : "none")};
   transition: border 200ms ease, transform 200ms ease, box-shadow 200ms ease;
 
-  &:hover,
-  &:focus {
+  ::placeholder {
```

Comment:
@axross Why did you delete & ?

Reply:
`&` refers itself. And referring itself is automatically completed by nesting. You need it only when using `& > .something` or `&.something`.